### PR TITLE
NIAD-445: added wiremock sandbox, removed swagger, amend deployment s…

### PR DESCRIPTION
**Pull request by Alicja-Kainos**
…cripts

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: ecs-proxies-deploy-sandbox.yml [hunk 1]
```diff
@@ -6,11 +6,6 @@ docker_service:
...
   - name: sandbox
     expose: true
-    environment:
-      - name: NODE_ENV
-        value: production
-      - name: LOG_LEVEL
-        value: "{{ 'debug' if APIGEE_ENVIRONMENT == 'internal-dev' else 'info' }}"
     health_check:
       matcher: "200"
       path: "/_ping"
```
Comment (petervdm) [inline ecs-proxies-deploy-sandbox.yml:11]: This should be /healthcheck

File: ecs-proxies-deploy.yml [hunk 2]
```diff
@@ -15,6 +15,8 @@ docker_service:
         value: ou=services,o=nhs
       - name: MHS_SDS_REDIS_CACHE_HOST
         value: cache-service
+      - name: MHS_SDS_REDIS_DISABLE_TLS
```
Comment (petervdm) [inline ecs-proxies-deploy.yml:18]: Where did this come from?  Please can you remove it.

File: ecs-proxies-deploy-sandbox.yml (hunk 1 above)
Comment (Alicja-Kainos) [reply ecs-proxies-deploy-sandbox.yml]: corrected

File: ecs-proxies-deploy.yml (hunk 2 above)
Comment (Alicja-Kainos) [reply ecs-proxies-deploy.yml:18]: Hmm, I think it is some merging effect, I saw it in conflict and thought it is something necessary

Comment (Alicja-Kainos) [reply ecs-proxies-deploy.yml]: removed

**Pull request merged**